HW04.Rmd

```diff
@@ -29,33 +18,70 @@ Knitr is part of the knitr package. Allows you to change column names and add ca
 
 
 ```{r month, warning=FALSE}
+gun_deaths_per_month <- thedata %>%
+    select(month) %>%
+    group_by(month) %>%
+    count(month) %>%
+    rename(deaths_per_month = n)
 
+kable(gun_deaths_per_month, align = 'c', col.names = c("Month", "Deaths/Month"))
 ```
 
 
 ### Generating a bar chart with human-readable labels on the x-axis. That is, each month should be labeled "Jan", "Feb", "Mar" (full or abbreviated month names are fine), not `1`, `2`, `3`.
 
 ```{r month_chart}
+thedata$month <- factor(thedata$month, labels = c("January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"))
 
+ggplot(thedata, aes(x = month, fill = month)) +
+    geom_bar() + 
+    labs(x = "Month", y = "NUmber of Deaths") 
 ```
+## Is there a way to make my axis text smaller so they're not overlapping like this? Also, I'm going to try and make the graph slightly less colorful. As it stands I think it's a bit too flamboyant. 
 
 ## Generating a bar chart that identifies the number of gun deaths associated with each type of intent cause of death. The bars should be sorted from highest to lowest values.
 
 ```{r intent}
-
+intent_graph <- thedata %>%
+    group_by(intent) %>%
+    count() %>%
+    rename(number_of_deaths = n)
+## Going back to give kable another shot, it seems to be working better for me than ggplot (previous figure)
+## Also, would you guys happen to know a way to order these rows by ascending/descending n? I'm curious if there's a function to do that automatically without having to manually adjust it ourselves.

```

Comment:
When creating my bar chart I used fct_infreq( ) to reorder the data.  This reorders by number of observations at each level with the largest coming first, so in descending order.  This is where I read about the function (https://forcats.tidyverse.org/reference/fct_inorder.html) and there are a few others to note that also help with reordering data.

Reply:
This is really helpful, thank you! I'll be checking out the reference a lot more now to help with my assignments